src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Markup_InCodeBlocksWithTagHelper.cshtml

```diff
@@ -0,0 +1,47 @@
+﻿@addTagHelper "*, TestAssembly"
+@{
+    var people = new Person[]
+    {
+        new Person() { Name = "Taylor", Age = 95, },
+    };
+
+    void PrintName(Person person)
```

Comment:
`Task PrintName(Person person)`.

The screenshot in this PR showcases the error that @rynowak was talking about where TagHelper usage inside of a void method don't notify the user of the non-asyncness at design time 😄

Reply:
We can try to address that later. Do we have an issue for that?

Reply:
Now we do https://github.com/aspnet/AspNetCore/issues/8630